### PR TITLE
Add Chat Bylaws

### PR DESCRIPTION
**Pull request by snazy**
Introduce the `CHAT_BYLAWS.md` file, as [discussed on the dev@ mailing list](https://lists.apache.org/thread/09qqj7xqs9q3vd9ho2z518wqkormrhp2).

### CODE REVIEW

**Review by dimas-b — APPROVED** [review]

**Review by eric-maynard — APPROVED** [review]
LGTM, but maybe this doesn't need to be a top-level file?

File: CHAT_BYLAWS.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* 3rd parties who contribute to or use the Polaris project may ask the (P)PMC to get a dedicated public channel, for
+  example #vendor-<vendor-name> or #project-<project-name>.
+* Promotion of 3rd party content is only permitted in these 3rd party channels, and only if the content primarily covers
+  Polaris. Commercial advertisements in any form are prohibited.
+* Interaction with users of 3rd party OSS projects or commercial products should be directed to these 3rd party
+  channels.
+
+## Content moderation
+
+* User content in any form is not moderated.
```
Comment (RussellSpitzer) [inline CHAT_BYLAWS.md:47]: To simplify this I would recommend we do a slight rewording

1. Any content which violates the Apache CoC will be removed
2. Any commercial advertisements will be removed
3. All other content will be unmoderated

**Review by RussellSpitzer — DISMISSED** [review]

**Comment by dbosco** [comment]
How does the discussions and decisiongs in dev emails and this channel reconcile?

**Comment by snazy** [comment]
> How does the discussions and decisiongs in dev emails and this channel reconcile?

It's currently being voted on.

File: CHAT_BYLAWS.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Apache Polaris - Project Chat Bylaws
```
Comment (jbonofre) [inline CHAT_BYLAWS.md:1]: I would include ASF header here (just to be clean as packaged in the distribution).

File: CHAT_BYLAWS.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+* Promotion of 3rd party content is only permitted in these 3rd party channels, and only if the content primarily covers
+  Polaris. Commercial advertisements in any form are prohibited.
+* Interaction with users of 3rd party OSS projects or commercial products should be directed to these 3rd party
+  channels.
+
+## Content moderation
+
+* Any content which violates the Apache “Code of Conduct” will be removed.
+* Any commercial advertisements will be removed.
+* All other content will not be moderated.
```
Comment (jbonofre) [inline CHAT_BYLAWS.md:49]: I would rephrase like `All other content is allowed as soon as it complies with the CoC`.

File: CHAT_BYLAWS.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+* The Polaris project’s chat tool is not provided by the ASF - the Polaris project (P)PMC members govern and monitor the
+  chat service.
+* Everybody is welcome to join the Polaris project chat. Invites are not needed.
+* All users are automatically promoted to “members” (don’t stay in “guests”).
+* Polaris project (P)PMC members have “owner” privileges on the Zulip chat. Polaris project committers are granted
+  “moderator” or “administrator” privileges for moderation purposes.
+* This bylaws document shall be published on the project’s web site and linked from relevant public channels.
+* Only (P)PMC members are allowed to create new channels. The number of channels shall be limited to #general (user
+  discussions), #dev (development discussions), #announcements (release announcements), and #noise (GitHub feed).
+* Only (P)PMC members are allowed to notify a large number of users (aka @here).
```
Comment (jbonofre) [inline CHAT_BYLAWS.md:30]: I would add an advise here to "invite" people to summarize important discussion on the dev mailing list.

File: CHAT_BYLAWS.md (hunk 3 above)
Comment (snazy) [reply CHAT_BYLAWS.md:49]: That's what the first bullet point already states?

File: CHAT_BYLAWS.md (hunk 4 above)
Comment (snazy) [reply CHAT_BYLAWS.md:30]: We can always direct people to go to the dev-ML. "Important" is something blurry - neither 100% objective nor subjective.

**Review by jbonofre — APPROVED** [review]
